docs/guide/ecosystem/client-server-communication.md

```diff
@@ -12,7 +12,7 @@ Axios is a famous library for performing async requests. It has a very nice and
 
 You can directly import Axios from NPM and use it in your components, without the need to install a dedicated plugin.
 
-However another pattern you are more likely to implement is to write a "client" where you create an Axios instance. Not only does this increase testability of your application, but it also allows you to centralise HTTP settings (Base URL and port for instance).
+However another pattern you are more likely to implement is to write a "client" where you create an Axios instance. Not only does this increase testability of your application through the use of mocking/stubbing, but it also allows you to centralise HTTP settings (Base URL and port for instance).
```

Comment:
```suggestion
However another pattern you are more likely to implement is to write a "client" or a "service" where you create an Axios instance and use that instead of just importing axios everywhere. 
Not only does this make your application easier to test, through the use of mocking/stubbing, but it also allows you to centralise HTTP settings (Base URL and port for instance).
```